pkgs/kde/misc/glaxnimate/default.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  mkKdeDerivation,
+  fetchFromGitLab,
+  qttools,
+  potrace,
+  ffmpeg,
+  libarchive,
+  python3Packages,
+  testers,
+  glaxnimate,
+  xvfb-run,
+}:
+
+mkKdeDerivation rec {
+  pname = "glaxnimate";
+  version = "0.5.80";
+
+  src = fetchFromGitLab {
+    domain = "invent.kde.org";
+    owner = "graphics";
+    repo = "glaxnimate";
+    tag = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-+4vvp9nxtpKUOojgQL9T5Eyv9eMCGGwmDAex91XPwyA=";
+  };
+
+  extraBuildInputs = [
+    qttools
+    potrace
+    ffmpeg
+    libarchive
+    # Has vendored `qt-color-widgets` and `pybind11`.
+  ];
+
+  qtWrapperArgs = [
+    "--prefix PYTHONPATH : ${
+      python3Packages.makePythonPath [
+        python3Packages.pillow
+        python3Packages.lottie
+      ]
+    }"
+  ];
+
+  passthru.tests.version = testers.testVersion {
+    package = glaxnimate;
+    command = "${lib.getExe xvfb-run} glaxnimate --version";
```

Comment:
This can 100% be versionCheckHook.

Reply:
FWIW

Reply:
It actually 100% can’t, I think: `versionCheckHook` can only handle one argument, which isn’t enough when you need `xvfb-run`. (This is copied from the old package, FWIW.)

Reply:
It probably doesn't need the xvfb-run either, but whatever, we can sort this out later.